docs: Link to new Discord

nickmccurdy opened the pull request.

kentcdodds commented on docs/contributing.md:
```diff
@@ -32,7 +32,7 @@ For questions related to using the library, please visit a support community
 instead of filing an issue on GitHub.
 
 - [Spectrum][spectrum]
```
Let's delete this

kentcdodds commented on website/core/Footer.js:
Let's delete spectrum everywhere

kentcdodds commented on website/pages/en/support.js:
```diff
@@ -28,7 +28,7 @@ function Support(props) {
       title: 'Spectrum',
```
Here too

nickmccurdy commented:
I've removed most of Spectrum links, but I'm running into a layout issue involving the support links in Docusaurus, which I'm not very experienced with. When removing a link, the two rows no longer have three items each, and I get this unnatural layout where the second row is different.
![image](https://user-images.githubusercontent.com/927220/85207199-24a8df00-b2f5-11ea-886f-56531b9e156c.png)


One thing I've noticed is that we're giving more than three items to `<GridBlock layout="threeColumn" align="left" />`, which is only supposed to take three. However, even if I split this into two `GridBlock`s, I can't seem to get the second row to be left aligned.

kentcdodds commented:
I'm not sure 😬

nickmccurdy commented:
@alexkrolick Would you mind taking a look at this issue if you have time?

kentcdodds commented:
Meh, let's just merge it. We can fix it later.

alexkrolick commented:
@nickmccurdy sorry I missed this, yeah, I think we got lucky with the layout aligning in the right number of columns. I recall trying a few different combos (like center-alignining different things, and different column counts) but the layout never really looked that great. Feel free to change it up.
